### PR TITLE
[10.x] received transfers

### PR DESCRIPTION
**Pull request by gkmk**
Add received transfers function for v10

### CODE REVIEW

File: src/Models/Wallet.php [hunk 1]
```diff
@@ -162,4 +164,17 @@ protected function initializeMorphOneWallet(): void
...
     }
+
+    /**
+     * returns all the receiving transfers to this wallet.
+     *
+     * @return HasMany<Transfer>
+     */
+    public function receivedTransfers(): HasMany
+    {
+        return app(CastServiceInterface::class)
```
Comment (rez1dent3) [inline src/Models/Wallet.php:175]:
```php
        return $this->hasMany(config('wallet.transfer.model', Transfer::class), 'to_id')
```

Comment (rez1dent3) [reply src/Models/Wallet.php:175]: @gkmk fix pls^

**Comment by rez1dent3** [comment]
Thanks again for your help with the project. Thanks for the pull request.

**Comment by gkmk** [comment]
Its very good looking code base :) happy to contribute

**Pull request merged**